src/Zicht/Bundle/PageBundle/Manager/PageManager.php

```diff
@@ -1,32 +1,68 @@
 <?php
 /**
- * @author Gerard van Helden <gerard@zicht.nl>
  * @copyright Zicht Online <http://zicht.nl>
  */
 
 namespace Zicht\Bundle\PageBundle\Manager;
 
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Persistence\ObjectRepository;
 use Doctrine\Common\Util\ClassUtils;
 use Doctrine\ORM\Mapping\ClassMetadata;
-use Doctrine\Bundle\DoctrineBundle\Registry;
+use Symfony\Component\EventDispatcher\Event as SymfonyEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Zicht\Bundle\PageBundle\Entity\ViewablePageRepository;
 use Zicht\Bundle\PageBundle\Event;
+use Zicht\Bundle\PageBundle\Model\PageInterface;
 use Zicht\Util\Str;
-use Zicht\Bundle\PageBundle\Entity\ViewablePageRepository;
 
 /**
  * Main service for page management
  */
 class PageManager
 {
-    protected $mappings = array();
-    protected $loadedPage = null;
+    /**
+     * @var array
+     */
+    private $mappings = array();
```

Comment:
- `[]`. 
- Perhaps initialize it in the constructor instead of here. (Other arrays are initialized there as well.)